Formula/gitwatch.rb

```diff
@@ -0,0 +1,34 @@
+class Gitwatch < Formula
+  desc "Watch a file or folder and automatically commit changes to a git repo easily"
+  homepage "https://github.com/gitwatch/gitwatch"
+  url "https://github.com/gitwatch/gitwatch/archive/refs/tags/v0.1.tar.gz"
+  sha256 "ed52d5c799c19dc6f920f3625964bc4b4948b6f8929c289aece462304e419697"
+  license "GPL-3.0-or-later"
+  head "https://github.com/gitwatch/gitwatch.git"
+
+  depends_on "coreutils"
+  depends_on "fswatch"
+
+  def install
+    bin.install "gitwatch.sh" => "gitwatch"
+  end
+
+  test do
+    temprepo = testpath/"temprepo"
+    tempfile = temprepo/"testfile"
+    system "git init #{temprepo}"
+    pid = fork do
+      exec "gitwatch -s 1 -m 'Auto commit' #{temprepo}"
+    end
+    Process.setpgid pid, pid
+    system "sleep 5"
+    system "touch #{tempfile}"
+    system "sleep 20"
+    begin
+      gitlog = shell_output("git --git-dir #{temprepo}/.git log -n 1 | head -1 | awk '{print $1}'").strip
```

Comment:
Parse this using Ruby [String](https://ruby-doc.org/core-2.6.3/String.html) methods instead of piping to `head` and `awk`.

Reply:
Fixed.